containers/concent-api/run-celery.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+
+cd "/srv/http/concent_api"
+
+exec "/srv/http/virtualenv/bin/celery" worker    \
+    --app=concent_api                            \
+    --loglevel=INFO
```

Comment:
Please rebase this on master. I think we added parameters to this script in another pull request.

Reply:
Done.